packages/graphqlgen/src/generators/flow-generator.ts

```diff
@@ -2,7 +2,11 @@ import * as os from 'os'
 import * as prettier from 'prettier'
```

Comment:
This module contains bug-fix code in response to the new compilation step we do.